qiskit_quantuminspire/qi_jobs.py

```diff
@@ -52,16 +77,96 @@ def __init__(
             if isinstance(run_input, QuantumCircuit)
             else [CircuitExecutionData(circuit=circuit) for circuit in run_input]
         )
+        self.program_name = "Program created by SDK"
+        self.batch_job_id: Union[int, None] = None
```

Comment:
You allow the user to supply a job_id (presumably referring to batch_job_id), but you set it to None here either way?